### ISSUE TITLE
Hm.. Possibility of porting this over to Vivaldi/Chrome/Opera?

### ISSUE THREAD

**Issue by NovaViper** [Quicksaver/OmniSidebar#151]
Hey, I really, **REALLY** enjoyed this extension on Firefox and I was completely heart-broken to find that it was going away in Firefox 57. I **literally** dumped Firefox completely soon after (curse you WebExtensions!) and have been in limbo hoping between Chrome, Opera, and very recently, Vivaldi. I used to be a die-heart Firefox fan, I hated using anything that was related to Chrome but until I noticed my many favorite plugins cease to function as to the changes to the extensions store V_V. So.. (if you want to of course, I understand that you wouldn't because of the technicalities and such) would such a beautiful extension like this **ever** be available on these Chromium based browsers?

**Comment by Quicksaver** [comment]
Unfortunately probably not, at least not made by me. :(

**Comment by NovaViper** [comment]
@Quicksaver Awh, maybe someone (or me when I get the chance, I still have school) will go and port it over to the Chromium browsers. I wish to thee farewell 😞

**Comment by lollox** [comment]
Why should you port OmniSidebar on Vivaldi? It already HAS a damn sidebar toggle and even a vertical "toolbar" to choose what panel to show... exactly as in the old glorious Opera and as it should be in EVERY browser

(shame on idiots who don't use sidebars)

**Comment by NovaViper** [comment]
I know it has one but, I fancy the way OmniSidebar functions. Vivaldi doesn't feature side menu like OmniSidebar did

**Comment by lollox** [comment]
Well, in every case consider that from Firefox 57 there will be no more browsers (PaleMoon apart) that will support UI changes through extensions, so asking for an extension that interact/change the UI is (sadly) useless ;)

**Comment by Verulam** [comment]
Bookmark Sidebar (https://extensions.blockbyte.de/) is a pretty good extension for Chrome, with much of OmniSidebar's functionality to display bookmarks in a hideable 'pop-out' sidebar. As I will be going over to Chrome when FF57 is released, and have found OmniSidebar indispensable, I was very pleased to discover it.

**Comment by NovaViper** [comment]
@lollox Yeah.. its unfornate for that. However, I THINK Vivaldi might consider moving away from Chromium once they get a stable foundation of users.

**Comment by NovaViper** [comment]
@Verulam Actually, that extension is nearly the same thing as Vivaldi's sidebar bookmarks (coincidence, right?). I just wish I could have **two** sidebars on Vivaldi

**Comment by Verulam** [comment]
Nova,

Does the Vivaldi sidebar autohide?

**Comment by NovaViper** [comment]
@Verulam It doesn't actually
